docs/extensions/ScrollViewerExtensions.md

```diff
@@ -55,18 +76,65 @@ class DoubleTopThicknessConverter : IValueConverter
 }
 ```
 
-The converter is used to just bind to top margin, moving only the ScrollBar's top end.
+## MiddleClickScrolling
+
+MiddleClickScrolling allows you to scroll by click middle mouse button (scroll wheel button) and move the pointer of the direction to be scrolled. This extension method can be used directly in `ScrollViewer` or ancestor of `ScrollViewer`.
+
+### Syntax
+
+```xaml
+<!-- Setting MiddleClickScrolling directely for ScrollViewer -->
+<ScrollViewer extensions:ScrollViewerExtensions.EnableMiddleClickScrolling="True">
+    <!-- ScrollViewer Content -->
+</ScrollViewer>
+
+<!-- Setting MiddleClickScrolling fot the ancestor of ScrollViewer -->
+<ListView extensions:ScrollViewerExtensions.EnableMiddleClickScrolling="True">
+    <!-- ListView Item -->
+</ListView>
+```
+
+### Sample Output
+
+![MiddleClickScrolling](../resources/images/Extensions/MiddleClickScrolling.gif)
+
+### Changing Cursor Type
+
+> [!IMPORTANT]
+Resource file must be manually added to change the cursor type when middle click scrolling. If you didn't add then the cursor won't change when middle click scrolling but funcanilaty won't be affected.
+
+#### Using Existing Resource File
+
+1. Download [MiddleClickScrolling-CursorType.res](https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.UI/Extensions/ScrollViewer/MiddleClickScrolling-CursorType.res) file
+2. Move this file into your project's folder
+2. Open .csproj file of your project in [Visual Studio Code](https://code.visualstudio.com/) or in any other code editor
+3. Added `<Win32Resource>MiddleClickScrolling-CursorType.res</Win32Resource>` in the first `<PropertyGroup>`
+
+### Using Your Own Resource File
```

Comment:
Is there documentation that we can link to for creating custom resource files?

Reply:
There is an [old doc for Windows 8](https://blogs.msdn.microsoft.com/devfish/2012/08/01/customcursors-in-windows-8-csharp-metro-applications/) but it is not necessary. We can just edit the existing resource file in Visual Studio.